ssl/t1_trce.c

```diff
@@ -1500,7 +1500,7 @@ void SSL_trace(int write_p, int version, int content_type,
             int hvers;
 
             /* avoid overlapping with length at the end of buffer */
-            if (msglen < (SSL_IS_DTLS(ssl) ? 13 : 5)) {
+            if (msglen < (size_t)(SSL_IS_DTLS(ssl) ? 13 : 5)) {
```

Comment:
While your touching this line anyway it would be good to replace 13 with `DTLS1_RT_HEADER_LENGTH` and 5 with `SSL3_RT_HEADER_LENGTH` rather than have the hard coded constants.

Reply:
I can do that easily. give me a minute.